### PR TITLE
fix: #299697: Repeat play a bar, goes back to first bar briefly

### PR DESCRIPTION
**Pull request by jpear1**
Resolves: https://musescore.org/en/node/299697

Made cursor rewind earlier in loop play mode and changed rewind target from 0 to loopIn.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Under which name did you sign the CLA?

**Comment by jpear1** [comment]
Vivalande. I tried to change my username to jpear1 but I think it may be
taking a while to update.

**Comment by ProbablyFaiz** [comment]
Looks good to me, fixed the bug on my machine

File: mscore/seq.cpp [hunk 1]
```diff
@@ -833,8 +833,10 @@ void Seq::process(unsigned framesPerPeriod, float* buffer)
                         if (mscore->loop()) {
                               int loopOutUTick = cs->repeatList().tick2utick(cs->loopOutTick().ticks());
                               if (loopOutUTick < scoreEndUTick) {
-                                    // Also make sure we are not "before" the loop
-                                    if (playPosUTick >= loopOutUTick || cs->repeatList().utick2tick(playPosUTick) < cs->loopInTick().ticks()) {
+                                    qreal framesPerPeriodInTime = static_cast<qreal>(framesPerPeriod) / MScore::sampleRate;
+                                    signed framesPerPeriodInTicks = cs->utime2utick(framesPerPeriodInTime);
```
Comment (Spire42) [inline mscore/seq.cpp]: The MuseScore codebase doesn't use `signed` for `int`. This should be either `int` or `auto`.

Comment (jpear1) [reply mscore/seq.cpp]: PR has been updated.

**Comment by anatoly-os** [comment]
@jpear1 why do you make a PR from your account but using a branch of another person (https://github.com/CCS-1L-F19)?

**Comment by jpear1** [comment]
I was working on MuseScore as part of a class project and the professor asked that all of our repos be in the class organization so that he could easily track everyone's progress. I can submit another PR from my account there's worries about the CLA agreement.

**Comment by Jojo-Schmitz** [comment]
Yes, better resubmit or rebase to come from your account

**Comment by jpear1** [comment]
[Done](https://github.com/musescore/MuseScore/pull/5941).

**Pull request closed without merging**